subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -1737,15 +1959,23 @@ uint16_t ull_conn_lll_max_tx_octets_get(struct lll_conn *lll)
 		 * Deduct 10 bytes for preamble (1), access address (4),
 		 * header (2), and CRC (3).
 		 */
+#if defined(CONFIG_BT_LL_SW_SPLIT_LLCP_LEGACY)
```

Comment:
Here a version a la ll_length_default_set() with a #define selecting either lll->max_tx_time or lll->dle.eff.max_tx_time could have been neater, but this works too.

Reply:
Yes, but i wanted to change as little as possible in the existing code, so that's why it was done as so